rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2021-10-29"
```

Comment:
This isn't the version we use ci. This downloads its in every ci run.

Reply:
Yeah, I checked the CI logs to check for the version being used but blundered and copied the cargo nightly release date instead of rustc.

```
nightly-x86_64-unknown-linux-gnu (overridden by +toolchain on the command line)
rustc 1.58.0-nightly (46b8e7488 2021-11-07)

[32;1m$ cargo +nightly --version[0;m
cargo 1.58.0-nightly (94ca096af 2021-10-29)
```